lib/libsys/intro.2

```diff
@@ -511,8 +512,13 @@ system call was issued on a socket, pipe or FIFO.
 An attempt was made to modify a file or directory
 on a file system that was read-only at the time.
 .It Er 31 EMLINK Em "Too many links" .
-Maximum allowable hard links to a single file has been exceeded
-.Pq limit of 32767 hard links per file .
+Maximum allowable hard links to a single file has been exceeded.
+This limit is a filesystem dependent variable
+.Po
+.Va UFS_LINK_MAX No on Xr ufs 4 ,
+.Va UINT32_MAX No on Xr fusefs 4 , and
+.Va INT_MAX No on Xr tmpfs 4
```

Comment:
I think it's very for error(2), which this page is mlinked to.
I'm honored to move it anywhere else if you find that appropriate, imho this is a nice detail for the manual. It already contained an outdated specifier, so I was thinking of this as maintenance rather than adding anything new.